markdown/developers/modules.md

```diff
@@ -510,6 +510,14 @@ process {
 
 3. Optional inputs are not currently supported by Nextflow. However, passing an empty list (`[]`) instead of a file as a module parameter can be used to work around this issue.
 
+4. Optional outputs SHOULD be marked as optional:
+
+   ```nextflow
+   tuple val(meta), path('*.tab')                   , optional:true, emit: tab
```

Comment:
```suggestion
   tuple val(meta), path('*.tab'), emit: tab,  optional: true
```
This is just a style suggestion. I prefer the emits to line up and spaces

Reply:
Sure, I just copy/pasted directly from a module, but happy to adjust.